conf/utd_sysbio.config

```diff
@@ -0,0 +1,43 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'University of Texas at Dallas HPC cluster profile provided by nf-core/configs'
+  config_profile_contact = 'Edmund Miller(@emiller88)'
+  config_profile_url = 'http://docs.oithpc.utdallas.edu/'
+  singularity_cache_dir = '/scratch/applied-genomics/singularity'
+}
+
+env {
+    TMPDIR = '/home/$USER/scratch/tmp'
+}
+
+singularity {
+  enabled = true
+  envWhitelist='SINGULARITY_BINDPATH'
+  autoMounts = true
+  cacheDir = params.singularity_cache_dir
```

Comment:
This should maybe go in a pipeline specific config? Unless it's a new DSL2 template feature (I've not dived into that yet 😅)

Reply:
The singularity cache? Nah, that should work with dsl1 as well. Ideally it should allow everyone to use the same images, so we don't each have our own copies of them. In theory ;)

Reply:
Are you sure? I don't see it in the template [`nextflow.config`](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config) on neither `master` or `dev`, so I would assume it's something specific to a given pipeline? I've never seen it before anyway :grimacing: